_source/_posts/2021-11-08-angular-v13.md

```diff
@@ -0,0 +1,183 @@
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+
+    const spanEl = fixture.debugElement.query(By.css('span'));
+    expect(spanEl).toBeTruthy();
+    expect(spanEl.nativeElement.innerHTML).toEqual('Welcome stranger!');
+  });
+});
+```
+
+We can configure the teardown cleanup in this test suit by changing the value of `destroyAfterEach`. With this change enabled by default, you get the best option of writing new tests with better performance and disabling it for tests that aren't ready to consume the changes. Angular has our backs on non-breaking updates!
+
+## Enhanced developer tooling
+
+Creating Angular apps is now easier with enhancements to developer tooling, compilation time improvements, and updates to testbed environments.
+
+The first thing you might notice is how much faster the build time is. A lot of work went into improving the build time. By adding a build cache, the Angular team found build time improvements of **up to 65+%**!
+
+The Angular Language Service's tooling improvement to auto-apply optional chaining feels like a small thing but is such an appreciated feature. I can focus on the object property I want to use, and the tooling handles the optional chaining for me!
+
+{% img blog/angular-v13/auto-apply-optional-chaining.gif alt:"Auto applying optional chaining in VS Code for a deep object with optional properties" width:"400" %}{: .center-image }
+
+One thing to note about the demo shown above is that it is for demonstration purposes and exaggerates capabilities to showcase the new feature, so please remember to apply sound software development practices to your applications.
+ 
+## So long, Internet Explorer!
+
+In this release, we also say goodbye to Internet Explorer. [Dropping Internet Explorer support](https://github.com/angular/angular/issues/41840) means lots of doors open up for new features, refactoring the code-base, and keeping up with security best practices. With extra polyfills, you can use Okta's okta-auth-js library in Internet Explorer, but the recommendation is to migrate away from it.
+
+## All the Angular v13 details
+
+These are the features that I found most interesting, but there's a lot more in Angular v13! You can read more about them in the [Angular team's blog post](https://blog.angular.io/angular-v13-is-now-available-cce66f7bc296).
+
+## Upcoming changes
+
+Angular keeps improving and growing. The [subsequent releases](https://angular.io/guide/roadmap) should prove exciting, with features such as strict typing for forms and stand-alone components in the works.
+
+Don't forget to follow us on [Twitter](https://twitter.com/oktadev) and subscribe to our [YouTube channel](https://www.youtube.com/c/OktaDev/) for more great content. What are your favorite Angular v13 features, or what updates are you looking forward to the most? Feel free to drop it in the comments below, as well as any questions or requests for tutorials you'd like to see next.
```

Comment:
We typically add links to previous Angular posts in this section. I'm not sure we need to here, but it might be worth having a few.